src/gui/settings.cpp

```diff
@@ -1192,42 +1192,74 @@ void FurnaceGUI::drawSettings() {
         ImGui::AlignTextToFramePadding();
         ImGui::Text("Arcade/YM2151 core");
         ImGui::SameLine();
-        ImGui::Combo("##ArcadeCore",&settings.arcadeCore,arcadeCores,2);
```

Comment:
Use tables. Put name of chip in one column and render/playback core combo boxes in the other two.